assets/toolboxIcon.svg

```diff
@@ -1,4 +1 @@
-<svg width="17" height="15" viewBox="0 0 336 276"
-  xmlns="http://www.w3.org/2000/svg">
-  <path d="M291 150V79c0-19-15-34-34-34H79c-19 0-34 15-34 34v42l67-44 81 72 56-29 42 30zm0 52l-43-30-56 30-81-67-66 39v23c0 19 15 34 34 34h178c17 0 31-13 34-29zM79 0h178c44 0 79 35 79 79v118c0 44-35 79-79 79H79c-44 0-79-35-79-79V79C0 35 35 0 79 0z"/>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-image"><rect x="3" y="3" width="18" height="18" rx="2" ry="2"></rect><circle cx="8.5" cy="8.5" r="1.5"></circle><polyline points="21 15 16 10 5 21"></polyline></svg>
```

Comment:
@Akash187 could you add a newline at the end of the file, thanks.

Reply:
Hello @MarioRodriguezS,
Thanks for testing my pull request. I should have tested before creating the merge request, I thought it will work as the change seems to be very small. Learned a lesson to always test no matter how small the change is.

I have fixed and tested the SVG. It is working fine. You can test as well.
Thank you
<img width="761" alt="Screenshot 2022-01-20 at 3 25 09 AM" src="https://user-images.githubusercontent.com/21134455/150220319-c8880878-6874-48e2-86a7-44c43f265cbb.png">

Reply:
@Akash187 Hi there! 
Yes, it is working :ok_hand:  , check the comments please.

Reply:
Thanks, I am adding a new line